nservicebus/azure-servicebus/publisher-names-configuration.md

```diff
@@ -0,0 +1,26 @@
+---
+title: Publishers name configuration
+summary: Configuration mapping between publisher names and message types for Endpoint Oriented Topology
+tags:
+- Cloud
+- Azure
+- Transports
+---
+
+`ITopology` interface exposes a property (`NeedsMappingConfigurationBetweenPublishersAndEventTypes`) to instruct transport environment if mapping must be configured. Custom topologies have to properly configure this property.   
```

Comment:
I've been thinking and wonder if we're not leaking some internals here. This is _only_ needed by the `EndpointOrientedTopology`. `ForwardingTopology` doesn't need it. We don't know what custom topologies will look like, so we're assuming they'll have to implement this. Feels off.